_posts/2020-10-02-snowplow-javascript-tracker-2-16-0-released/index.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: post
+title: "Snowplow JavaScript Tracker 2.16.0 released"
+title-short: Snowplow JavaScript Tracker 2.16.0
+tags: [snowplow, javascript, tracker]
+author: Paul
+category: Releases
+permalink: /blog/2020/10/02/snowplow-javascript-tracker-2.16.0-released/
+discourse: false
+---
+
+We are pleased to announce a new release of the [Snowplow JavaScript Tracker][js-tracker].
+
+[Version 2.16.0][2.16.0-tag] comes with a 30% file size reduction as we improve our build tooling in preparation for the upcoming major release v3.0.0. Specifically, v2.15.0 was 111KB (34KB when gzipped). The new v2.16.0 is a much improved 77KB (27KB when gzipped) while maintaining exactly the same functionality.
+
+Read on below for:
+
+1. [Switching build tooling](#1-switching-build-tooling)
+2. [Other features and bug fixes](#2-other-features-and-bugfixes)
+3. [Upgrading](#3-upgrading)
+4. [Documentation and help](#4-documentation-and-help)
+
+<!--more-->
+
+## 1. Switching build tooling
+
+As originally suggested by `bernardosrulzon`, we have switched to using the Closure Compiler ([GitHub issue #583](https://github.com/snowplow/snowplow-javascript-tracker/issues/583)). We have also switched to using rollup.js and gulp ([GitHub issue #752](https://github.com/snowplow/snowplow-javascript-tracker/issues/752)). Together, these two updates reduce the `sp.js` filesize by 30%. 
+
+## 2. Other features and bug fixes
+
+This tracker also comes with a few other under the hood improvements:
+
+* Switch to Github Actions ([GitHub issue #844](https://github.com/snowplow/snowplow-javascript-tracker/issues/844))
+* Add license banner to rollup builds ([GitHub issue #845](https://github.com/snowplow/snowplow-javascript-tracker/issues/845))
+* Bump snowplow-tracker-core to 0.9.2 ([GitHub issue #841](https://github.com/snowplow/snowplow-javascript-tracker/issues/841))
+* Update Snowplow Micro to v1 ([GitHub issue #837](https://github.com/snowplow/snowplow-javascript-tracker/issues/837))
```

Comment:
There's an opportunity to make a bigger deal out of using Snowplow Micro to test our own trackers here... perhaps plug the latest v1 release a little? Link off to the other blog post perhaps.